promql/promqltest/testdata/functions.test

```diff
@@ -77,7 +71,7 @@ load 5m
 # Tests for increase().
 eval instant at 50m increase(http_requests[50m])
 	{path="/foo"}   100
-	{path="/bar"}    90
+	{path="/bar"}   88.88888888888889
```

Comment:
The test was designed to yield a round result. Let's extend the range to 55m here to keep it that way.

Reply:
OK, I played with it. I still think we should change the test so that it yields round numbers, but that requires some more changes. Here is my suggestion:

```Go
# Tests for increase().
load 5m
	http_requests{path="/foo"}	0+10x11
	http_requests{path="/bar"}	0+10x5 0+10x5
	http_requests{path="/dings"}   10+10x11
	http_requests{path="/bumms"}    1+10x11

eval instant at 55m increase(http_requests[55m])
	{path="/foo"}   110
	{path="/bar"}    99
	{path="/dings"} 110
	{path="/bumms"} 110
```